tests/pylint/runpylint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python
```

Comment:
Hi Brian,
see your previous comment. Did you say 'python2' by mistake ? Anyway, I'm running this inside a virtualenv (both locally and in Travis CI) with Python 3.5. However the interpreter isn't always called 'python3' hence the change.
The only thing I can think off is using
```
if not six.PY3:
    sys.exit(1)
```

at the top of this script. How about the others ?